build.rs

```diff
@@ -84,7 +84,7 @@ fn main() {
 
     rsconf::rebuild_if_env_changed("FISH_GETTEXT_EXTRACTION_FILE");
 
-    cc::Build::new().file("src/libc.c").compile("flibc.a");
+    // cc::Build::new().file("src/libc.c").compile("flibc.a");
```

Comment:
> My primary motivation is to improve and simplify support for running fish shell on Windows 11 systems.

Not sure if this is easy because fish relies on lots of Unix APIs.
There are some closed issues that list potential problems.

But regardless, moving away from src/libc.c and into the libc crate
sounds like a good idea.